tests/infra/ccf.py

```diff
@@ -102,12 +102,13 @@ def _add_node(self, node, lib_name, args, target_node=None):
         )
 
         # If the network is opening, node are trusted without consortium approval
-        if self.status == ServiceStatus.OPENING and args.consensus != "pbft":
-            try:
-                node.wait_for_node_to_join()
-            except TimeoutError:
-                LOG.error(f"New node {node.node_id} failed to join the network")
-                raise
+        if self.status == ServiceStatus.OPENING:
```

Comment:
We need a way for operators to check the state of a node without the command to go through the consensus. As discussed, this could be a new flag in frontend's `install()` to install a handler as a "no-consensus" command. We should add a TODO here as well.